scss/chat/_chat-list-item.scss

```diff
@@ -187,10 +187,6 @@
   .mute_icon {
     background-color: var(--chatListItemSelectedText);
   }
-
-  .status-icon.sending {
-    background-color: white;
-  }
 }
```

Comment:
Don't remove stuff that affects other themes.
you can do instead in your theme:
```scss
.chat-list-item, .pseudo-chat-list-item {
 .chat-list-item.selected {
   .status-icon.sending {
     background-color: unset;
   }
 }
}
```

Reply:
Oh shit, I didn't want to remove that one, only the draft icon.